src/transformStateWithClones.js

```diff
@@ -8,6 +8,32 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const states = [];
+  let copyStates = { ...state };
```

Comment:
I think would be better to rename it.
It is actually but one state version, so it doesn't have to be in plural in order to be clear to understand
I would come up with sth like this:

```suggestion
  let stateVersion = { ...state };
```